Ryujinx.Graphics.Gpu/Image/TextureGroup.cs

```diff
@@ -331,24 +359,55 @@ public void SynchronizeMemory(Texture texture)
         /// <param name="regionCount">The number of handles to synchronize</param>
         private void SynchronizePartial(int baseHandle, int regionCount)
         {
+            int spanEndIndex = -1;
+            int spanBase = 0;
+            ReadOnlySpan<byte> span = ReadOnlySpan<byte>.Empty;
+
             for (int i = 0; i < regionCount; i++)
             {
                 if (_loadNeeded[baseHandle + i])
                 {
                     var info = GetHandleInformation(baseHandle + i);
 
+                    if (spanEndIndex <= i - 1)
+                    {
+                        spanEndIndex = i;
+
+                        if (_is3D)
+                        {
+                            // Look ahead to see how many handles need loaded.
```

Comment:
```suggestion
                            // Look ahead to see how many handles need loaded.
```
"need loaded" sounds bit weird. Maybe "needs to be loaded"?